bodhi/tests/server/test_notifications.py

```diff
@@ -201,99 +79,79 @@ def test_no_messages(self):
     def test_clear_messages_on_send(self):
         """Assert the message queue is cleared after the event handler runs."""
         session = Session()
-        session.info['messages'] = [api.Message()]
+        session.info["messages"] = [api.Message()]
 
         with fml_testing.mock_sends(api.Message()):
             notifications.send_messages_after_commit(session)
 
-        self.assertEqual(session.info['messages'], [])
+        self.assertEqual(session.info["messages"], [])
```

Comment:
This seems like an unrelated style change.

I'll stop pointing these out, but let's make the commits atomic and avoid style changes.